src/include/btree.i

```diff
@@ -945,12 +915,14 @@ __wt_page_release_busy(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
 	 */
 	locked = WT_ATOMIC_CAS4(ref->state, WT_REF_MEM, WT_REF_LOCKED);
 	WT_TRET(__wt_hazard_clear(session, page));
-	if (!locked)
+	if (!locked) {
+		WT_TRET(EBUSY);
```

Comment:
That was an intentional change, but not based on any testing.  I thought we should sleep if the page is already locked by another thread...  I'll back it out and see if it helps with the column store truncate hangs.